opt_types.go

```diff
@@ -0,0 +1,26 @@
+package opts
+
+import "fmt"
+
+type RepeatedStringOpt struct {
```

Comment:
Instead of this type, I was going to support multiple flags by doing `[]type`, might get rid of `commalist` `spacelist`

Reply:
This was more an example of providing a custom type implementing flag.Value than an alternative to `opt:"[]type"` (not sure exactly what you're thinking of doing).

Should files an issue, multi opts doesn't currently error, the last one wins.

Reply:
So `[]type` as in

```
type Config struct {
  Strs []string
  Ints []int
}
```

just magically works (multi-flag).

Reply:
Added some helper types https://github.com/jpillora/opts/blob/subcommands/util.go